lib/spack/spack/cmd/create.py

```diff
@@ -57,8 +57,7 @@ class {class_name}({base_class_name}):
     """FIXME: Put a proper description of your package here."""
 
     # FIXME: Add a proper url for your package's homepage here.
-    homepage = "http://www.example.com"
-    url      = "{url}"
+    homepage = "http://www.example.com"{url}
```

Comment:
I assume `{url}` is appended directly here to avoid introducing an extra newline when there is no URL, IMO it's fine to add space for the sake of readability.

Reply:
Yes, that was the reason.  

I was trying to avoid excess space in case people were using static analysis tools; however, if that were the case, there are a lot of packages with issues.